### PR TITLE
FIX: sslTime Not an int32 value

### PR DESCRIPTION
**Pull request by qinhang3**
FIX: TS2339: Property 'includes' does not exist on type 'string'.
FIX FOR CHROME(https://bugs.chromium.org/p/chromium/issues/detail?id=404501): If a resource is fetched using a pre-existing HTTPS connection then secureConnectionStart is set to zero when it should really be a timestamp.

Resolves https://github.com/apache/skywalking/issues/5783

### CODE REVIEW

**Comment by wu-sheng** [comment]
@JTrust @zhangkewei Please check.

File: src/performance/perf.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ class PagePerf {
         // Synchronous load resources in the page
         resTime: parseInt(String(timing.loadEventStart - timing.domContentLoadedEventEnd), 10),
         // Only valid for HTTPS
-        sslTime: location.protocol.includes('https') ?
+        sslTime: location.protocol === 'https' && timing.secureConnectionStart > 0 ?
```
Comment (arugal) [inline src/performance/perf.ts:51]: image

@qinhang3  It looks like `https:`, please confirm :)

Comment (qinhang3) [reply src/performance/perf.ts]: You are right.. 😊

**Review by arugal — APPROVED** [review]
image

LGTM

**Pull request merged**